Web UI/CognitiveSearch.UI/wwwroot/css/site.css

```diff
@@ -375,17 +376,22 @@ pre {
 }
 
 .results-body {
-    padding-left: 0;
+    padding-left: 20px;
```

Comment:
Couple of points here:
- it's good to be consistent with using units, for ex. since most of the elements are using pixels, it might be good to change the padding-bottom and margin-bottom to use pixels as well
- for adding multiple padding styles, it is better to use the `padding` shorthand property, makes the code a lot more readable, and reduces the lines in the css file too. For ex. this would be rewritten as `padding: 0px 0px 100px(whatever the correct value for 100% is) 20px`

Here's the doc. on reading up re: this, 
https://www.w3schools.com/css/css_padding.asp